sdk/eventhub/azure-eventhub/azure/eventhub/_transport/_pyamqp_transport.py

```diff
@@ -461,6 +461,24 @@ def create_mgmt_client(
             connection_verify=config.connection_verify,
         )
 
+    @staticmethod
+    def open_mgmt_client(mgmt_client, conn):
+        """
+        Opens the mgmt AMQP client.
+        :param AMQPClient mgmt_client: pyamqp AMQPClient.
+        :param conn: Connection.
+        """
+        try:
+            mgmt_client.open(connection=conn)
+        except FileNotFoundError as exc:
+            # TODO: added for uamqp error parity with invalid connection_verify path
+            # consumer raises FileNotFoundError, producer raises EventHubError
+            # consumer opens connection through mgmt client first, producer through SendClient
+            # need to remove in the future
+            if exc.filename:
+                exc.filename2 = "consumer"
```

Comment:
only consumer raises FileNotFoundError for invalid connection_verify scenario, and encounters this error when opening mgmt client. producer raises EventHubError, and encounters the error when opening the send client. adding to distinguish and raise the correct error.

Reply:
removing this and passing in a `is_consumer` kwarg from mgmt_request method in client bases. Adding a comment in the code there to clarify.